Scripts/build-archive.sh

```diff
@@ -27,6 +27,26 @@ if [ ! -d "$PRODUCTS_DIR/iOS/AppCenterDistributeResources.bundle" ] || \
   exit 1
 fi
 
+# Verify prefix.
+verify_prefix() {
+
+  # Get values.
+  frameworkPath=$1
+
+  # Get result from command and check result value.
+  if nm -gU $frameworkPath | awk '{ print $3 }' | grep "_OBJC_CLASS_" | grep -v "_OBJC_CLASS_\$_MSAC"; then
+    echo "These files should have a prefix MSAC."
+    exit 1
```

Comment:
`exit` is not the right behaviour here. Use `return` (or even just use the only one command from this function to print issues for all frameworks.